mozilla/fx-private-relay
# MPP-2417 - Phone Welcome Screen

This adds the Phone Welcome Screen step prior to launching the user to the phone dashboard upon completing onboarding.

**How to test:**
Go to the phones page,
1) The resend welcome sms button should disappear upon being clicked, and generate a toast notification.
2) Select 'Go to dashboard', and it should take you to the phone dashboard.

For both instances, a cookie is set to determine whether the resend sms button / welcome screen has been observed yet by the user. 

https://user-images.githubusercontent.com/13066134/202303291-2cabc972-f04b-4b3f-9cc6-d45208fcf3f0.mov



https://user-images.githubusercontent.com/13066134/202304632-a06ee4c2-3e24-4549-b756-a4eddeae307a.mov



# Checklist
